core/domain/question_services.py

```diff
@@ -465,6 +564,7 @@ def _save_question(committer_id, question, change_list, commit_message):
     question_model.language_code = question.language_code
     question_model.question_state_data_schema_version = (
         question.question_state_data_schema_version)
+    question_model.linked_skill_ids = list(set(question.linked_skill_ids))
```

Comment:
I think we should not be doing conversion here, to avoid skew between the domain object and the storage model. Shouldn't question.linked_skill_ids already have unique elements anyway?

Reply:
Done